settings-kits.gradle

```diff
@@ -31,7 +31,8 @@ include (
         ':kits:googleanalyticsfirebase-kit',
         ':kits:pilgrim-kit',
         ':kits:onetrust-kit',
-        ':kits:adobemedia-kit'
+        ':kits:adobemedia-kit',
+        ':kits:neura-kit'
```

Comment:
this shouldn't be added here as you will be hosting the kit in your own repo, releasing it, etc

Reply:
No problem, removed